lib/data-builder.js

```diff
@@ -67,8 +67,10 @@ function fillDebugData(data, err) {
     if ((p in data)) continue;
     data[p] = err[p];
   }
-  // NOTE err.stack is not an enumerable property
+  // NOTE `err.stack` and `err.message` is not an enumerable property
   data.stack = err.stack;
+  if (err.message)
```

Comment:
if we do `data.message = err.message;`
the result will come back with 
`"message" : ""`